File: cpp/05_functions.cpp
#include <iostream>
#include <string>
using namespace std;

string greet(string name) {
    return "hello " + name;
}

int add(int a, int b) {
    return a + b;
}

int main() {
    cout << greet("ulas") << endl;
    cout << add(3, 5) << endl;
    return 0;
}

File: csharp/05_functions.cs
using System;

class Program {
    static string Greet(string name) {
        return "hello " + name;
    }

    static int Add(int a, int b) {
        return a + b;
    }

    static void Say(string msg = "hi") {
        Console.WriteLine(msg);
    }

    static void Main() {
        Console.WriteLine(Greet("ulas"));
        Console.WriteLine(Add(3, 5));
        Say();
        Say("yo");
    }
}
